SPECS/libsmi/libsmi.spec

```
@@ -1,23 +1,24 @@
-Vendor:         Microsoft Corporation
-Distribution:   Mariner
+Summary:        A library to access SMI MIB information
 Name:           libsmi
 Version:        0.4.8
-Release:        27%{?dist}
-Summary:        A library to access SMI MIB information
-
-License:        GPLv2+ and BSD
-URL:            http://www.ibr.cs.tu-bs.de/projects/libsmi/index.html
-Source0:        ftp://ftp.ibr.cs.tu-bs.de/pub/local/libsmi/%{name}-%{version}.tar.gz
+Release:        28%{?dist}
+License:        TCL AND BSD-3-Clause
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://www.ibr.cs.tu-bs.de/projects/libsmi/index.html
+#Upstream Source0 url is dead
+Source0:        %{_mariner_sources_url}/%{name}-%{version}.tar.gz
 Source1:        smi.conf
-Source2:	IETF-MIB-LICENSE.txt
-Patch0:		libsmi-0.4.8-wget111.patch
-Patch1:		libsmi-0.4.8-CVE-2010-2891.patch
-Patch2:		libsmi-0.4.8-symbols-clash.patch
-Patch3:		libsmi-0.4.8-format-security-fix.patch
-
+Source2:        IETF-MIB-LICENSE.txt
+Patch0:         libsmi-0.4.8-wget111.patch
+Patch1:         libsmi-0.4.8-CVE-2010-2891.patch
```

Let's re-name this patch, since we're here, to make sure our CVE scanning tool can find it:
```suggestion
Patch1:         CVE-2010-2891.patch
```

Naturally, this will also require a file re-name, but that's a small thing.